packages/site/src/data/angular/libraries.ts

```diff
@@ -23,555 +23,609 @@ export const libraryTags = [
 	"testing",
 ] as const
 
+// The default image here for use
+// on libraries that don't have an image
+const defaultImage = "https://github.com/angular.png"
+
 export const libraries: Library<typeof libraryTags[number]>[] = [
 	{
 		name: "Angular CLI",
 		author: "Google",
-		gitHubRepo: "angular/angular-cli",
-		npmPackage: "@angular/cli",
+		repo: "https://www.github.com/angular/angular-cli",
+		package: "https://www.npmjs.com/package/@angular/cli",
 		href: "https://angular.io/cli",
 		description:
 			"The Angular CLI is a command-line interface tool that you use to initialize, develop, scaffold, and maintain Angular applications directly from a command shell",
-		image: "https://github.com/angular.png",
+		image: defaultImage,
 		tags: ["framework"],
+		language: libraryTags[0],
```

Comment:
My concern here is that if a contributor decides to alter the library tags array and adds a tag at the top, then that will affect the ordered here. It might be better to a string here

Reply:
Sounds good! I'll swap those over tomorrow! 😄